src/tests/Directory.Build.targets

```diff
@@ -271,11 +271,135 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ConvertCountOfStressMarkersToListPhase1" BeforeTargets="ConvertCountOfStressMarkersToListPhase1">
+    <PropertyGroup>
+      <SequenceOfSemicolons>$([System.String]::new(';',10))</SequenceOfSemicolons>
+    </PropertyGroup>
+  </Target>
+
   <Target Name="GenerateMarkerFiles" BeforeTargets="AssignTargetPaths">
+    <PropertyGroup Condition="'$(NumberOfStripesToUseInStress)' == '' and '$(IsMergedTestRunnerAssembly)' == 'true'">
+      <NumberOfStripesToUseInStress>1</NumberOfStripesToUseInStress> <!-- Set a custom number of stripes in stress to allow very large merged groups to function in the presence of GCStress -->
+    </PropertyGroup>
+
+    <!-- Generate a set of stress marker files to be used in stress scenarios in CI runs -->
```

Comment:
Could we add a comment describing how the number of number of <code>SequenceOfIntegers</code> items correlates to the stripe width and total test count as a guidance for future engineers trying to tweak these magic constants? (Nit - for brevity I'd consider rolling several values, e.g. 10, into the same item line using the semicolon separator but that's just my personal feeling, I don't insist and in a way your formulation is more straightforward.)

Reply:
Ah, so what happens here is that this is used to do some math. There is no correlation here at all, other than the number of stripes must be less than or equal to the maximum number here.  I'm writing a small markdown file describing various project file options for use in our test infra, which will describe what a developer can actually use.

Reply:
Would it be perhaps possible to use your md as the nucleus of the doc I mentioned above i.e. a more general explanation of how merged test runners work, what environment variables and command line options are relevant and so on? I'm certainly not saying you should write all of that, I'm just pointing it out as it might affect your choice of location and naming for the file.